kolibri/plugins/video_mp4_render/kolibri_plugin.py

```diff
@@ -11,8 +11,7 @@ class VideoMP4RenderPlugin(KolibriPluginBase):
 
 class VideoMP4RenderAsset(webpack_hooks.WebpackBundleHook):
     unique_slug = "video_mp4_render_module"
-    src_file = "kolibri/plugins/video_mp4_render/assets/src/module.js"
-    static_dir = "kolibri/plugins/video_mp4_render/static"
+    src_file = "assets/src/module.js"
```

Comment:
When we're not explicit about where the build output ends up.. then where does it end up? :)